Swiftfin/Views/SettingsView/SettingsView.swift

```diff
@@ -83,6 +83,23 @@ struct SettingsView: View {
 				}
 			}
 
+			Section(header: L10n.quickConnect.text) {
```

Comment:
Instead of making another section in the main settings page, make another `Button` in the first section that routes to a view to handle Quick Connect. You can see how this is done for many other views in `SettingsCoordinator`.